Source/DataTypes/SvgViewBox.cs

```diff
@@ -156,10 +156,10 @@ public void AddViewBoxTransform(SvgAspectRatio aspectRatio, ISvgRenderer rendere
                     fScaleX = Math.Min(fScaleX, fScaleY);
                     fScaleY = Math.Min(fScaleX, fScaleY);
                 }
-                float fViewMidX = (this.Width / 2) * fScaleX;
-                float fViewMidY = (this.Height / 2) * fScaleY;
-                float fMidX = width / 2;
-                float fMidY = height / 2;
+                float fViewMidX = (this.Width / 2);
+                float fViewMidY = (this.Height / 2);
+                float fMidX = width / 2 * fScaleX;

```

Comment:
thanks for the pull request, but what are these changes? they don't seem to be related to .NET 3.5. there was recently a pull request regarding the same problem. please remove this file from the pull request, update your repo to the latest state, test again and make a separate PR for this issue.